pkg/controller/appdefinition/testdata/volumes/named/existing.yaml

```diff
@@ -0,0 +1,12 @@
+---
+kind: ClusterVolumeClassInstance
+apiVersion: internal.admin.acorn.io/v1
+metadata:
+  name: custom-class
+description: Just a simple test volume class
+default: true
+storageClassName: custom-class
+minSize: 1Gi
```

Comment:
Just realizing it down here - Should we be looking at Gigabytes instead of Gibibytes for VolumeClasses since its storage?

Reply:
Either works, as long as the validation works out.